setuptools/command/editable_wheel.py

```diff
@@ -811,7 +823,7 @@ def install():
         sys.path_hooks.append(_EditableNamespaceFinder._path_hook)
     if PATH_PLACEHOLDER not in sys.path:
         sys.path.append(PATH_PLACEHOLDER)  # Used just to trigger the path hook
-"""
+'''
```

Comment:
We probably can skip this change in the quotes right? Does black complain about it?

Reply:
Yes, I think so - this was so the docstring in `_in_dir_case_sensitive` could use triple-double quotes.

Reply:
Ah, sorry, I did not see that.